### DISCUSSION TITLE
WIFI_PROTOCOL_LR mode

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#905] [Q&A] [answered]
my Esp32 is in close box so may be that's why i getting good result within few meters only . 

so i am planning for espnow long  with WIFI_PROTOCOL_LR mode . 

does it will have any negative effect with current API  ? ?

**Comment by aqildad-create** [comment]
I observe strange behavior after using  ESP_ERROR_CHECK( esp_wifi_set_protocol(WIFI_IF_STA, WIFI_PROTOCOL_LR )); 
NOW my esp32 is not connecting to internet any more..  Even after  uploading basic code of esp32 wifi connectivity but still its not connecting .

 
i try with above with 2 esps and both have same results . 

 I know its not related to your great APi but i discus it here because i am confuse what wrong with these line of code .

**Reply by pschatzmann** [reply, marked as answer]
This is outside of my scope
